doc/release-notes/8578-support-extended-iso-639-languages.md

```diff
@@ -0,0 +1,16 @@
+The Controlled Vocabulary Values list for the metadata field Language in the Citation block can be extended to include more ISO 639 languages. This process will merge new values into the existing list adding more than 400 additional languages. For best results please load the new citation.tsv file first as this merge will re-order the list in the display based on the 'Value' and 'identifier' fields.
+Any Controlled Vocabulary Values can be extended by using the tag #controlledVocabularyExt instead of the standard #controlledVocabulary tag. Using this tag will ensure the original alternate values will not be deleted, items with equal unique identifier ids, and/or alternate values, will be merged, and the display order will be regenerated based on the display name ('Value') sorted alphabetically with rows having no identifier at the end of the list (i.e. value:'Not Applicable' id:'').
+
+Example of merged language:
+
+` citation.tsv: Dutch nld dut nl`
+
+` language_ISO_639.tsv: Dutch, Flemish dut nld nl`
+
+` Updated in DB as: Dutch, Flemish nld dut nl`
+
+```
+wget https://raw.githubusercontent.com/IQSS/dataverse/v6.3/scripts/api/data/metadatablocks/language_ISO_639.tsv
+curl http://localhost:8080/api/admin/datasetfield/load -H "Content-type: text/tab-separated-values" -X POST --upload-file language_ISO_639.tsv
```

Comment:
Why do we have a separate language_ISO_639.tsv file instead of putting the extra languages in citation.tsv? Are we worried some installations won't want all the extra languages?

For someone reading the Installation Guide, how would they discover that extra languages are available in the language_ISO_639.tsv file? Can we have something in source/installation/config.rst?

Reply:
There were concerns about the size of the list. Also, this feature allows installations the ability to merge new tsv files and re-order the list as opposed to adding new values to the end of the list. The Ext tag can be used by other CVV types and not just languages.

I will update the installation guide

Reply:
Looks like the merging of languages isn't going to work. I see there is a citation.properties file which I believe hold the translations for the language name to the local language. This file would not be touched so the merged languages would not be found. Merged languages would always default to English representations in the list.

Reply:
No longer relevant